ag_generation.py

```diff
@@ -3,8 +3,14 @@
 from signatures.mappings import macro_inv, micro, micro2macro, verbose_micro
 
 
-# Translate technical nodes to human-readable
 def _translate(label, root=False):
+    """
+    Translates the node label into a more human-readable version.
+
+    @param label: the label of the node (`mcat|mserv`, where mcat is an integer)
```

Comment:
mcat is an integer mapped from a dictionary? State the dictionary name as well for clarity.

Reply:
Right, this was a mistake from my side, sorry. `mcat` was actually a string. Thanks for letting me know! I have updated the doc